files/justfiles/common/utilities.just

```
@@ -459,3 +459,16 @@ setup-usbguard:
     if ! rpm-ostree status | grep '●' | grep -q 'securecore'; then
         systemctl enable --user --now usbguard-notifier.service
     fi
+
+# Create the admin user and remove wheel from the normal user
+create-admin:
+    #!/bin/bash
```

Add `set -euo pipefail` to the beginning of the script so it will fail rather than continuing if any individual command fails.

---

Resolved in d34a6b7